CHANGELOG.md

```diff
@@ -10,9 +10,20 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 - **CUMULUS-2631**
   - Added 'Bearer token' support to s3credentials endpoint
-  
+
 ### Changed
 
+- **CUMULUS-3024**
+  - Update PUT /granules endpoint to operate consistently across datastores
+    (PostgreSQL, ElasticSearch, DynamoDB). Previously it was possible, given a
+    partial Granule payload to have different data in Dynamo/ElasticSearch and PostgreSQL
+  - Given a partial Granule object, the /granules update endpoint now operates
+    with behavior more consistent with a PATCH operation where fields not provided
+    in the payload will not be updated in the datastores.
+  - If a `null` files key is provided in an update payload (e.g. `files: null`),
+    an error will be thrown. `null` files were not previously supported and would throw potentially unclear errors. This makes the error clearer and more explicit.
+  - If an empty array is provided for the `files` key, all files will be removed in all datastores
```

Comment:
I don't want to spell out all of the logic here but I think it's important to highlight what's different in this case.

Reply:
https://github.com/nasa/cumulus/pull/3082/commits/7b5a6b064483f0d46ab1e3d6b442d90e41b11ba3